lib/puppet-lint/plugins/check_trailing_comma.rb

```diff
@@ -1,89 +1,86 @@
 PuppetLint.new_check(:trailing_comma) do
   def array_indexes
-    @array_indexes ||= Proc.new do
+    @array_indexes ||= proc {
```

Comment:
Doesn't puppet-lint expose these now? Perhaps first submit a PR to bump to puppet-lint 3.0 and use those new functions, then add Rubcop?

Reply:
I've removed `hash_indexes` and made changes to that `array_indexes` can be removed.

I'm going to hit `default_indexes` next.